lib/smart_answer/node.rb

```diff
@@ -22,6 +24,10 @@ def template_name(template_name = nil)
       @template_name
     end
 
+    def presenter(flow_presenter, state)
```

Comment:
Is it worth defining default parameters instead of passing empty arguments? (e.g. `presenter(nil, nil)` → `presenter()`)
```
    def presenter(flow_presenter ||= nil, state ||= nil)
```

Reply:
Good shout! I've added a commit to change the method to use kwargs and default values